cwms-data-api/src/test/java/cwms/cda/api/MeasurementControllerTestIT.java

```diff
@@ -0,0 +1,654 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2024 Hydrologic Engineering Center
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package cwms.cda.api;
+
+import cwms.cda.api.enums.UnitSystem;
+import static cwms.cda.data.dao.DaoTest.getDslContext;
+import cwms.cda.data.dao.DeleteRule;
+import cwms.cda.data.dao.MeasurementDao;
+import cwms.cda.data.dao.StreamDao;
+import cwms.cda.data.dto.CwmsId;
+import cwms.cda.data.dto.measurement.Measurement;
+import cwms.cda.data.dto.stream.Stream;
+import cwms.cda.formatters.ContentType;
+import cwms.cda.formatters.Formats;
+import static cwms.cda.security.KeyAccessManager.AUTH_HEADER;
+import fixtures.CwmsDataApiSetupCallback;
+import fixtures.TestAccounts;
+import static io.restassured.RestAssured.given;
+import io.restassured.filter.log.LogDetail;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.sql.SQLException;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.http.HttpServletResponse;
+import mil.army.usace.hec.test.database.CwmsDatabaseContainer;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.AfterAll;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import static org.hamcrest.Matchers.*;
+
+@Tag("integration")
+final class MeasurementControllerTestIT extends DataApiTestIT {
+
+    private static final String OFFICE_ID = TestAccounts.KeyUser.SPK_NORMAL.getOperatingOffice();
+    private static final List<Stream> STREAMS_CREATED = new ArrayList<>();
```

Comment:
Consider naming these lists something like `STREAMS_TO_CLEANUP` to clarify their purpose.

Reply:
Renamed to TEST_STREAMS. The streams get created, used, and then cleaned up for test purposes, so its more than just cleanup.